.github/workflows/appsec.yml

```diff
@@ -122,7 +122,7 @@ jobs:
   express:
     runs-on: ubuntu-latest
     env:
-      PLUGINS: express|body-parser|cookie-parser|multer
+      PLUGINS: express|body-parser|cookie-parser|multer|handlebars|pug
```

Comment:
I think we should not place `handlebars` and `pug` plugin tests in the `express` job.

These tests are not related w/ express, since they are not testing the express integration (they don't do `app.set('view engine', 'pug')` or `app.set('view engine', 'hbs')`). A separate job for these plugin test would be more suitable.

WDYT?

Reply:
I previously created separate jobs for `handlebars` and `pug`, but I encountered an error related to express, the only solution I found was to run them with express.
I tried something like `PLUGINS: express|handlebars|pug`  but that ends up running all the express tests again.

I've just pushed my previous commit, and you can see that the `AppSec / template` job is failing.

Reply:
I think the error is not related with express.

In the plugin test, the lib to be tested (hbs, pug) are [required](https://github.com/DataDog/dd-trace-js/pull/4827/files#diff-45d723a65b1a18f2cc2513af6b2acfebad56c4aa9c626be1cf38d22cb4f4925aR12) in the before fn, before calling the method `prepareTestServerForIast`. This method initializes everything necessary to be able to test the plugin, among other things, it initializes the tracer itself, which registers the hook of the require of the library, where we instrument these.

What is happening is that a require of the library is being done before the initialization of the tracer, and therefore, the first test fails because that library is not instrumented (you can see that the second test for the same library but a different version does pass).

The error would be solved by making the library require inside a `beforeEach` inside the `prepareTestServerForIast` cb.

Reply:
Like in this case: https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/test/appsec/iast/analyzers/sql-injection-analyzer.sequelize.plugin.spec.js#L27

Reply:
Thanks for this clear explanation, Im gonna fix it